db/schema.rb

```diff
@@ -53,11 +53,14 @@
     t.index ["slug"], name: "index_communities_on_slug", unique: true, using: :btree
   end
 
-  create_table "index_choices", force: :cascade do |t|
-    t.integer  "index_id"
-    t.integer  "choice_id"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+  create_table "indices", force: :cascade do |t|
+    t.datetime "finalized_at"
+    t.integer  "category_id"
```

Comment:
well, we would have to set the foreign_key option changed it to `awrvi_version`, and in fact we probably would want to do something like `awrvi_version_id` and a `belongs_to :awrvi_version, class_name: 'Category'`